src/util/container-request.js

```diff
@@ -0,0 +1,51 @@
+import {detectContainerForBytes, getId3Offset} from '@videojs/vhs-utils/dist/containers';
+import {stringToBytes, toUint8} from '@videojs/vhs-utils/dist/byte-helpers';
+import {callbackWrapper} from '../xhr';
+
+const containerRequest = (uri, xhr, cb) => {
+  let bytes;
+  let id3Offset;
+
+  const progressListener = function(error, request) {
+    if (error) {
+      return cb(error, request, '', bytes);
+    }
+    const currentLength = bytes && bytes.length || 0;
+    const contentPart = request.responseText.substring(currentLength, request.responseText.length);
+    const newBytes = toUint8(stringToBytes(contentPart, true));
+    const temp = new Uint8Array(currentLength + newBytes.length);
```

Comment:
Might be good to use a more specific name, or specify why a temp var is being used.

Reply:
added a comment